Chatbot.py

```diff
@@ -3,14 +3,17 @@
 from streamlit_chat import message
 
 with st.sidebar:
-    openai_api_key = st.text_input('OpenAI API Key',key='chatbot_api_key')
+    openai_api_key = st.text_input(
+        "OpenAI API Key", key="chatbot_api_key", type="password"
+    )
```

Comment:
I still don't like doing all of these expansions in these examples since they were intended to be compact (although they have drifted pretty far from that already hah)

Reply:
Removed the expansions